app/src/routes/api/v1/webshot.router.js

```diff
@@ -17,7 +17,7 @@ const router = new Router({
 });
 
 const viewportDefaultOptions = { width: 1024, height: 768, isMobile: true };
-const gotoOptions = { waitUntil: 'networkidle' };
+const gotoOptions = { waitUntil: 'networkidle2' };
```

Comment:
can you provide more details on what exactly this does?

Reply:
We have upgraded the version of `puppeteer` and now the library has two options for waiting:

- `networkidle0` - Consider navigation to be finished when there are no more than 0 network connections for at least 500 ms.
- `networkidle2` - Consider navigation to be finished when there are no more than 2 network connections for at least 500 ms.

Previously, we only had `networkidle`, which considered navigation to be finished when the network activity stays "idle" for at least networkIdleTimeout ms (default 1000ms). I think that `networkidle2` is very similar to the previous one in the sense that we wait for the middle time on two connections instead of one